ISSUE_TEMPLATE.md

```diff
@@ -1,22 +1,9 @@
 <!--
-Thank you for contributing to Calypso!
-
-Docs & troubleshooting:
-=======================
-https://github.com/Automattic/wp-calypso/blob/master/CONTRIBUTING.md
-https://github.com/Automattic/wp-calypso/blob/master/docs/troubleshooting.md
-
-Think you found a bug?
-======================
-Use the template below to log a new bug. If you want to prefix the title with a “Question:”, “Bug:”, or the general area of the application, that would be helpful, but by no means mandatory. If you have write access, add the appropriate labels.
-
-Have a feature request?
-=======================
-If you want to prefix the title with “Question:”, “Feature Request:”, or the general area of the application, that would be helpful. For example, "Editor: add ability to choose from most popular tags" would make a great title.
+Thank you for contributing to Calypso! This template is to guide you along, so you can delete it if you wish. This text won't appear once you submit your issue.
 -->
 
 <!-- BUG TEMPLATE -->
```

Comment:
@stephanethomas Yes, because the feedback was that it was confusing and people weren't sure if they could delete it or if it was optional.